lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -4354,7 +4354,7 @@ namespace Js
             AssertMsg(false, "Non-matching start of document");
         }
 
-        GetLineCharOffsetFromStartChar(cchStartOffset, &line, &col, false /*canAllocateLineCache*/);
+        GetLineCharOffsetFromStartChar(cchStartOffset, &line, &col, false /*canAllocateLineCache*/, true /*canDumpLibraryCode*/);
```

Comment:
you should probably pass `canDumpLibraryCode` directly instead of hardcoding the true value here. In release builds, Asserts are compiled out and won't hit. We want to ensure we still get the intended (and reasonable) behavior even if the Assert didn't cause execution to stop.